### PR TITLE
Add capi yolov8 ensemble pipeline profile

### PR DESCRIPTION
**Pull request by jim-wang-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [x] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [x] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**499**

## Test Instructions if applicable

- git clone this PR
- build capi-yolov8-ensemble pipeline profile: `make build-capi_yolov8_ensemble`
- start camera-simulator: `make run-camera-simulator`
- run pipeline: `PIPELINE_PROFILE="capi_yolov8_ensemble" DEVICE="CPU" RENDER_MODE=1 sudo -E ./run.sh --platform core inputsrc rtsp://127.0.0.1:8554/camera_0`
- run unit tests: ` make unit-test-profile-launcher` should see ALL PASSED
- run smoke tests should see ALL PASSED: ` make run-smoke-tests`

### CODE REVIEW

File: download_models/model_build/convert-model.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (antoniomtz) [inline download_models/model_build/convert-model.py:1]: Do we need to add copyright headers to new files?

**Review by antoniomtz — CHANGES_REQUESTED** [review]
Tested it and runs fine. Just one request

File: download_models/model_build/convert-model.py (hunk 1 above)
Comment (jim-wang-intel) [reply download_models/model_build/convert-model.py]: added the copyright headers

File: configs/opencv-ovms/envs/capi_yolov8_ensemble.env [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+GST_VAAPI_DRM_DEVICE=/dev/dri/renderD128
+TARGET_GPU_DEVICE=--privileged
+NO_BASH_C=1
+INPUT_TYPE=RTSP_H264
+LOG_LEVEL=0
+RENDER_MODE=1
+cl_cache_dir=/home/intel/gst-ovms/.cl-cache
+WINDOW_WIDTH=1280
+WINDOW_HEIGHT=720
+DETECTION_THRESHOLD=0.7
```
[the remark this answers is not in the document]
Comment (jim-wang-intel) [reply configs/opencv-ovms/envs/capi_yolov8_ensemble.env]: changed back to 0.5

**Review by antoniomtz — APPROVED** [review]

**Review by brian-intel — APPROVED** [review]

**Pull request merged**